### DISCUSSION TITLE
Digital/lab1/FAQ/

### DISCUSSION THREAD

**Discussion by giscus** [wintermelon008/USTC-DigitalLab-Discussion#10] [Announcements]
# Digital/lab1/FAQ/

https://soc.ustc.edu.cn/Digital/lab1/FAQ/

**Comment by hanquanjushi** [comment]
请问什么时候能在bb发布作业提交窗口

**Reply by wintermelon008** [reply]
我们正在确定 BB 上的提交方式，发布后会在群里通知大家~

**Reply by hanquanjushi** [reply]
Thanks

**Comment by iBug** [comment]
关于本页面的一些吐槽：

1. <s>大部分图片实际上不需要点开查看，也就是无需使用 `[![alt](link)](link)` 的语法，直接 `![alt](link)` 就够了。</s> 原来你们还整合了 glightbox，阅读体验非常棒👍
2. 额外添加的 CSS 为 `#my_table` 钦点了 `font-family`，并且 Arial 比较难看，建议删除字体设定，使用主题提供的 Roboto 字体。
3. 时间日期要么用良好的中文（2023 年 9 月 28 日），要么用类似 ISO-8601 的格式（2023-09-28），现在又用空格又用点的格式（2023 09.28）看着非常不规范，而且感觉怪怪的。

**Comment by iBug** [comment]
思考：致谢真的有必要每个 Lab 分开写吗？

**Reply by wintermelon008** [reply]
最开始确实把致谢分开了，感觉不太好，后面发布时会删掉的（

**Comment by magichear** [comment]
小白提问：

1. Lab1的实验报告部分，“测试结果与分析”板块需要写吗？

2. 题目1 的资源使用不为零，但WNS值为1是正常的吗？

        Thanks.

**Reply by magichear** [reply]
纠正提问，WNS值为NA

**Reply by wintermelon008** [reply]
1. 这个目前不用写，因为还没有到测试与运行阶段
2. 是正常的，可以参考[这里](https://support.xilinx.com/s/question/0D52E00006hpohISAQ/%E5%B8%83%E7%BA%BF%E9%80%9A%E8%BF%87%E4%BD%86%E6%98%AFwns%E7%AD%89%E4%B8%BAna?language=en_US)
